src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -485,7 +530,11 @@ public override void CopyFrom(BindableObject other)
         /// <summary>
         /// Release instance.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <param name="disposing">
+        /// If disposing equals true, the method has been called directly or indirectly by a user's code. Managed and unmanaged resources can be disposed.
```

Comment:
It's nice to have this detailed description, but this is a standard method and the description could have been simpler.

Reply:
Done. I made it simpler.